easybuild/easyconfigs/g/GDAL/GDAL-2.2.0-intel-2017a-Python-2.7.13-HDF5-1.10.1-HDF-4.2.13.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
+
+name = 'GDAL'
+version = '2.2.0'
+hdf5_ver = '1.10.1'
+hdf_ver = '4.2.13'
+versionsuffix = '-Python-%%(pyver)s-HDF5-%s-HDF-%s' % (hdf5_ver, hdf_ver)
+
+homepage = 'http://www.gdal.org/'
+description = """GDAL is a translator library for raster geospatial data formats that is released under an X/MIT style
+ Open Source license by the Open Source Geospatial Foundation. As a library, it presents a single abstract data model
+ to the calling application for all supported formats. It also comes with a variety of useful commandline utilities for
+ data translation and processing."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://download.osgeo.org/gdal/%(version)s/']
+sources = [SOURCELOWER_TAR_XZ]
+patches = ['GDAL-%(version)s_fix-uchar-definition.patch']
+checksums = [
+    '0d4c326862e0f118e17418c042c2bcd037b25abd3fb198e1fc5d40b11a9fc8ea',
+    '12daef0f16ebe438aeedc00b9eb2d35fdc5d39decffb567953459492a2a0fb5f',
```

Comment:
When there's more than 1 checksums, we include a comment at the end of the line to mention the file (`--inject-checksums` does this automagically).

```python
checksums = [
    '0d4c326862e0f118e17418c042c2bcd037b25abd3fb198e1fc5d40b11a9fc8ea',  # gdal-2.2.0.tar.xz
    '12daef0f16ebe438aeedc00b9eb2d35fdc5d39decffb567953459492a2a0fb5f',  # GDAL-2.2.0_fix-uchar-definition.patch
]
```

If the line becomes too long (> 120 chars, like it does with the patch), put it above the checksum:

```python
checksums = [
    '0d4c326862e0f118e17418c042c2bcd037b25abd3fb198e1fc5d40b11a9fc8ea',  # gdal-2.2.0.tar.xz
    # GDAL-2.2.0_fix-uchar-definition.patch
    '12daef0f16ebe438aeedc00b9eb2d35fdc5d39decffb567953459492a2a0fb5f',
]
```

Reply:
@boegel On your first comment, I am not fully sure whether HDF cripples anything.  However the standard gdal's do not support HDF (only HDF5).  We need to make clear to users that this is a special version which does.